api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -145,7 +145,10 @@ message ExternalProcessor {
   // to the processing mode) if the timer expires before a matching response
   // is received. There is no timeout when the filter is running in asynchronous
   // mode. Default is 200 milliseconds.
-  google.protobuf.Duration message_timeout = 7;
+  google.protobuf.Duration message_timeout = 7 [(validate.rules).duration = {
+    lte {seconds: 3600}
+    gte {}
```

Comment:
nit: can the comment clarify what happens when the `message_timeout` is 0? The motivation is that IIUC it is a different behavior than that of `max_message_timeout`, so should probably explicitly called out.

Reply:
"message_timeout" specifies how long Envoy ext_proc filter will wait for the response.  0 is a valid number in terms of timeout. i.e, envoy will start a timer with 0 ms.  And the results is that timeout event will happen right away.

"max_message_timeout" actually is not related to "message_timeout".  "max_message_timeout" is related to "override_message_timeout" config here:
https://github.com/envoyproxy/envoy/blob/26b259c25024e4c5cf40fd6ad5ae627f7795fba2/api/envoy/service/ext_proc/v3/external_processor.proto#L188

It provides the upper bound of the "override_message_timeout". It also can disable "override_message_timeout" API by setting "max_message_timeout" to be zero. There are some comments about these two fields here:
https://github.com/envoyproxy/envoy/pull/25802#discussion_r1127799443

Reply:
done